src/UI/InputField/InputField.css

```diff
@@ -1,18 +1,78 @@
-input::placeholder {
-  font-size: 14px;
+.input-group-UI {
+  position: relative;
+  margin-top: 20px;
+  /* margin just for test */
 }
 
-.input-field {
-  width: 60%;
-  height: 50px;
-  flex-grow: 1;
-  padding-right: 10px;
+.input-field-UI {
+  border: 2px solid gray;
```

Comment:
You need use variables from src/styles/variables.scss.
Check this on another places such as color, font size, border radius, transition, @media and ...
Maybe you need new variables you can add

Reply:
fixed or at least tried to fix